src/test/java/uk/gov/companieshouse/presenter/account/controller/PresenterAccountControllerTest.java

```diff
@@ -54,12 +57,12 @@ void testHealthCheckEndpointSuccessResponse() {
     @DisplayName("Return 201 when valid presenter details is submitted")
     void testCreatePresenterAccountSuccessResponse() {
         final String id = "id";
-        when(presenterAccountService.createPresenterAccount(presenterRequest)).thenReturn(id);
+        when(presenterAccountService.createPresenterAccount(presenterAccountDetailsRequest)).thenReturn(id);
 
-        var response = presenterAccountController.createPresenterAccount(presenterRequest);
+        var response = presenterAccountController.createPresenterAccount(presenterAccountDetailsRequest);
         var header = response.getHeaders().getFirst("Location");
         assertThat(response.getStatusCode(), is(HttpStatus.CREATED));
-        assertTrue(header.contains(PRESENTER_ACCOUNT + id));
+        Assertions.assertTrue(header != null && header.contains(PRESENTER_ACCOUNT + id));
```

Comment:
The purpose of the `header != null` is not actually to check if header is null, but instead to act as  a protection against `NullPointerException`s.
This is because the `&&` operator "shortcircuits". This means when the first argument is false it won't execute the second since "and" requires both arguments to be true. In this case if `header != null` is false (`header` is null) it won't execute `header.contains(PRESENTER_ACCOUNT + id)`. Therefore, preventing a null pointer exception from calling the `contains` method on a null class.
It is functionally the same as 

```
if (header 1== null) {
    assertTrue(header.contains(PRESENTER_ACCOUNT + id));
} else {
    throw new AssertionError("Header is null");
}
```